indico/modules/events/editing/client/js/components/FileManager/util.js

```diff
@@ -66,14 +66,20 @@ export function uploadFiles(action, fileTypeId, acceptedFiles, uploadURL, dispat
       const uploadedFile = await uploadFile(acceptedFile, uploadURL, e =>
         dispatch(actions.progress(fileTypeId, tmpFileId, Math.floor((e.loaded / e.total) * 100)))
       );
-      dispatch(
-        action(fileTypeId, fileId, tmpFileId, {
-          filename: uploadedFile.filename,
-          uuid: uploadedFile.uuid,
-          claimed: false,
-          fileType: fileTypeId,
-        })
-      );
+
+      if (uploadedFile === undefined) {
```

Comment:
I'd rather make the function return `null` and check for that, instead of using an implicit `undefined` return value.